packages/cdktf/lib/complex-computed-list.ts

```diff
@@ -1,9 +1,13 @@
 import { Token } from "./tokens";
-import { ITerraformResource } from "./terraform-resource";
+import { IResolvable } from "./tokens/resolvable";
+
+export interface IInterpolatingParent {
```

Comment:
I think we would be explicit about marking types that implement this interface.
One benefit of doing so is that it makes the types consuming this interface more jsii friendly.

Reply:
Also, not my favorite name, but not sure what else to call it.